MultimodalQnA/docker_compose/intel/cpu/xeon/compose.yaml

```diff
@@ -113,6 +113,7 @@ services:
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
       LVM_ENDPOINT: ${LVM_ENDPOINT}
+      MAX_IMAGES: ${MAX_IMAGES:-1}
```

Comment:
What is the `:-1` doing?

Reply:
It makes `1` the default if the `MAX_IMAGES` env var is not set, similar to `${TAG:-latest}` on line 104